gensim/similarity_learning/HowToRunModels.md

```diff
@@ -0,0 +1,23 @@
+# Running Similarity Learning Models
```

Comment:
should be part of eval repo

Reply:
I thought it would help people understand the similarity learning module.
Moving it now.

Reply:
All important stuff can be placed as docstrings of model + additional tutorials (for better understanding)